.stylelintrc

```diff
@@ -51,7 +51,16 @@
   },
   "overrides": [
     {
-      "files": ["./src/button/*.scss"],
+      "files": [
+        "./src/alert/*.scss",
+        "./src/anchor-navigation/*.scss",
+        "./src/annotation-context/*.scss",
+        "./src/app-layout/*.scss",
+        "./src/area-chart/*.scss",
+        "./src/attribute-editor/*.scss",
+        "./src/autosuggest/*.scss",
```

Comment:
Since you're doing it by the letter, you could probably just do `"./src/a*/*.scss"` 😄

Reply:
You have a keen eye!